source/scss/_pf-masthead.scss

```diff
@@ -0,0 +1,78 @@
+/*
+Copyright 2016 Red Hat, Inc
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+//
+// Masthead
+// --------------------------------------------------
+
+.pf-masthead {
+  background-color: $pf-color-black-1000;
+  border-top: 3px solid $pf-color-blue-300;
+  color: $pf-color-black-300;
+  @include media-breakpoint-up(md) {
+    padding-left: ($grid-gutter-width-base / 2);
+  }
+}
+
+.pf-masthead__skip {
```

Comment:
I can't believe I missed that. ![bitmoji](https://render.bitstrips.com/v2/cpanel/8375314-170621439_1-s4-v1.png?transparent=1&palette=1&width=246)